docs/develop/api_for_vms.rst

```diff
@@ -0,0 +1,46 @@
+Api to communicate with VMS
+===========================
+The purpose of building this API was to provide information about the meetups
+in Portal to VMS so that their volunteers could participate in them. Presently,
+there are three fields of meetups which are sent to VMS via the API::
+
+1. Event Name
+2. Start Date
+3. Venue
+
+Users can send in a ``GET`` or a ``POST`` request to the specified `URL <localhost/meetup/api/v1/request_meetup_data/>`_ to access the meetup data.
+
+In case of a ``GET`` request, a list containing the details of all meetups
+in the ascending order of their dates will be returned.
+
+In case of a ``POST`` request, a list containing the details of all meetups
+greater than or equal to the posted date will be returned.
+
+Details of the API
+------------------
+
+1. Use Case : Send event details for volunteers to contribute accordingly.
+
+2. API Method : GET/POST
+
+3. URL Parameters : ``localhost/meetup/api/v1/request_meetup_data/``
+
+4. Request Body::
+
+      {
+        "Date” : “date after which all meetups are required (yyyy-mm-dd)”
+      }
+
+5. Response Body - Success::
+
+      {
+  “Event Name/Title” : “ --------”
+     “Start Date” : “-----------”
+     “Venue” : “------------------”
+      }
```

Comment:
As seen [here](https://github.com/abhi20161997/portal/blob/26e11e6a9ffbfff04206d76a99aaa5d1b513f31d/docs/develop/api_for_vms.rst), the Response Body - Success and Response Body - Error can be formatted nicely.